core/src/main/java/org/testcontainers/containers/wait/internal/InternalCommandPortListeningCheck.java

```diff
@@ -24,22 +24,23 @@
 
     @Override
     public Boolean call() {
-        String command = "true";
+        StringBuilder command = new StringBuilder("true");
 
         for (int internalPort : internalPorts) {
-            command += " && ";
-            command += " (";
-            command += format("cat /proc/net/tcp* | awk '{print $2}' | grep -i ':0*%x'", internalPort);
-            command += " || ";
-            command += format("nc -vz -w 1 localhost %d", internalPort);
-            command += " || ";
-            command += format("/bin/bash -c '</dev/tcp/localhost/%d'", internalPort);
-            command += ")";
+            command.append(" && ");
+            command.append(" (");
+            command.append(format("cat /proc/net/tcp* | awk '{print $2}' | grep -i ':0*%x'", internalPort));
+            command.append(" || ");
+            command.append(format("nc -vz -w 1 localhost %d", internalPort));
+            command.append(" || ");
+            command.append(format("/bin/bash -c '</dev/tcp/localhost/%d'", internalPort));
+            command.append(")");
         }
 
         Instant before = Instant.now();
         try {
-            ExecResult result = ExecInContainerPattern.execInContainer(waitStrategyTarget.getContainerInfo(), "/bin/sh", "-c", command);
+            ExecResult result = ExecInContainerPattern.execInContainer(waitStrategyTarget.getContainerInfo(), "/bin/sh", "-c", command.toString());
+            log.trace("Port listener check result code [{}] with stdout message [{}]", result.getExitCode(), result.getStdout());
             log.trace("Check for {} took {}", internalPorts, Duration.between(before, Instant.now()));
```

Comment:
Let's merge these two `TRACE` logs into something like this:
```suggestion
            log.trace("Check for {} took {}. Result code '{}', stdout message: '{}'", internalPorts, Duration.between(before, Instant.now()), result.getExitCode(), result.getStdout());
```